command/render.go

```diff
@@ -0,0 +1,87 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jrasell/levant/levant"
+)
+
+// RenderCommand is the command implementation that allows users to render a
+// Nomad job template based on passed templates and variables.
+type RenderCommand struct {
+	args []string
+	Meta
+}
+
+// Help provides the help information for the template command.
+func (c *RenderCommand) Help() string {
+	helpText := `
+Usage: levant render [options] TEMPLATE
+
+  Render a Nomad job template, useful for debugging.
+
+General Options:
+	
```

Comment:
Final thing; it looks like the Help is missing the -var-file variable description.

Reply:
Ah, must have removed that when I culled the other options.